### PR TITLE
Add `Tutorials` section to docs

### LINKED ISSUE #85: Add "Tutorials" section in the documentation

### LINKED ISSUE #86: Add a first tutorial for Poisson equation: Copy and translate from: https://gridap.github.io/Tutorials/stable/pages/t001_poisson/

### PR DESCRIPTION
**Pull request by Cmurilochem**
As discussed in #84, I am adding a new tutorial section to the docs and will start the translation from the previous Gridap material.

## Notes
Similarly to [PartitionedArrays.jl](https://github.com/fverdugo/PartitionedArrays.jl/blob/b211949cc5842f3d6d99f8452b9f0233aef963a5/docs/make.jl#L9), this first attempt uses the `Literate` package to convert the `jl` tutorial into a `md` page.

Closes #85 and closes #86.

### CODE REVIEW

File: docs/src/tutorials/poisson_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+```@meta
+EditURL = "../../poisson_tutorial.jl"
+```
```
Comment (fverdugo) [inline docs/src/tutorials/poisson_tutorial.md:3]: This looks like a auto generated file. It is better to remove it from the repo. Maybe also edit the .gitignore to skip this type of files.

Comment (Cmurilochem) [reply docs/src/tutorials/poisson_tutorial.md]: Thanks @fverdugo. As suggested, I added the generated tutorials `md` files to `.gitignore`.

**Comment by Cmurilochem** [comment]
@fverdugo. I finally managed to clean up a bit the [t001_poisson.ipynb](https://github.com/GalerkinToolkit/GalerkinToolkit.jl/blob/add_tutorials_to_docs/notebooks/t001_poisson.ipynb), adding the new plotting functions. Maybe we could discuss today any possible improvements. However, I would like to kindly request you help to edit and possibly add text to it as this would also be important to us to start understand the software internals.

**Comment by fverdugo** [comment]
PR #122 substitutes this one

**Pull request closed without merging**